### PR TITLE
Create university-of-bucharest-faculty-of-law.csl

### PR DESCRIPTION
**Pull request by AlexandruPanait1**
This is the requested style of citation and bibliography to be followed by students of University of Bucharest Faculty of Law in creating their Dissertation Work for all Masters programs (style used since 2018). I would like to contribute to the enlargement of Zotero Style Repository with this html code. Please note that I am a student that was asked by a teacher to create this html code for the sole reason that I remember some html coding from highschool. I created this code unofficially, because our University does not care about the digitalization process and the creation of tools in order to help students. They will not direct resources in creating another code and probably will tolerate the situation. I will speak with the Vice Dean about the existence of this code that will be added to Zotero Style Repository.

### CODE REVIEW

**Comment by AlexandruPanait1** [comment]
I see that there are some failures. When I used CSL Validator 1.0.2 strict (for CSL repository) everything was good. My html understanding is low, what should I change to the code? Can you explain to me like you talk to a child? Or if the problem is not so big, can you change it?

**Comment by AlexandruPanait1** [comment]
Thank you for help! I saw that you removed the contributor among many others. I will remove it form `summary` too.

**Comment by AlexandruPanait1** [comment]
Good afternoon, is there anything else that I have to do in order to post this style in Zotero Style Repository?

File: university-of-bucharest-faculty-of-law.csl [hunk 1]
```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded" default-locale="ro-RO">
+  <info>
+    <title>University of Bucharest Faculty of Law (Română)</title>
```
Comment (bwiernik) [inline university-of-bucharest-faculty-of-law.csl:4]:
```suggestion
    <title>University of Bucharest - Faculty of Law (Română)</title>
```

**Comment by bwiernik** [comment]
Thanks for your patience!

**Comment by AlexandruPanait1** [comment]
> Thanks for your patience!

Not a problem! I am glad to contribute!

**Pull request merged**